pkg/networkserver/networkserver.go

```diff
@@ -55,6 +55,9 @@ const (
 
 	// networkInitiatedDownlinkInterval is the minimum time.Duration passed before a network-initiated(e.g. Class B or C) downlink following an arbitrary downlink.
 	networkInitiatedDownlinkInterval = time.Second
+
+	// SchemaVersion is the Network Server database schema version. Bump when a migration is required.
+	SchemaVersion = 1
```

Comment:
In theory the schema is an implementation detail of the store, so this probably is meant to go to the `nsredis` package.